### PR TITLE
Add os_ratings role for registering resources in Cloudkitty

### PR DESCRIPTION
**Pull request by markgoddard**

### CODE REVIEW

**Comment by markgoddard** [comment]
Currently untested.

**Comment by MoteHue** [comment]
I've tested using this at UZH, alongside a WIP version of Max's openstack-config branch https://github.com/stackhpc/openstack-config/tree/cloudkitty-role-config 
Service and field mappings were applied without issue :)

File: roles/os_ratings/README.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+              - value: large
+                cost: 2.0
+                group: instance_uptime_flavor_id
+                type: flat
+        os_ratings_hashmap_service_mappings:
+          - service: image.size
+            cost: 0.1
+            group: volume_ceph
+            type: flat
+```
```
Comment (MaxBed4d) [inline roles/os_ratings/README.md:83]: Isn't this going to be redundant once this gets pushed as the way to configure and run the CloudKitty playbook is with the role via `openstack-config` & `openstack-config.yml`.

**Review by MaxBed4d — APPROVED** [review]
Apart from potential changes to the `README.md` this role seemingly works as intended with it working both in a private AiO and @MoteHue's production environment.

File: roles/os_ratings/README.md (hunk 1 above)
Comment (markgoddard) [reply roles/os_ratings/README.md:83]: Generally I like roles to provide an example in their readme. openstack-config is a separate project and you don't need to use it to use this collection or role. I don't think we even link to it from this collection.

**Comment by markgoddard** [comment]
Thanks @MoteHue and @MaxBed4d for testing

**Pull request merged**